run_everything.R

```diff
@@ -0,0 +1,55 @@
+
+# ============================================================================ #
+# Update data used for the forecasts
+# ============================================================================ #
+source("utils/get_us_deaths.R")
+
+
+
+# ============================================================================ #
+# load in function to prepare submissions
+# ============================================================================ #
+source("utils/format_submission.R")
+
+
+
+
+# update delays (not run)
+# source("rt-forecast/update-delays.R")
+
+
+# ============================================================================ #
+# Run all models, format submission, do plotting
+# ============================================================================ #
+
+
+target_dates <- seq(ymd('2020-05-15'), ymd(Sys.Date()), by = '1 day')
```

Comment:
I am not sure we want historic formating in the same script as the function required to update. This will just get longer and longer and repeatedly rerun the same thing. Suggest having an update script to produce a forecast that either just runs on the most up to data and lets the scripts handle everything or takes the date as an input. 

Doing the historic update somewhere else makes sense